### ISSUE TITLE
Add docs on how to properly download media_store for Mac users

### ISSUE THREAD

**Issue by aaronraimist** [spantaleev/matrix-docker-ansible-deploy#34]

**Comment by aaronraimist** [comment]
Or add a new doc that explains how to `rsync -a -P /path_to_backup/media_store/ /matrix/synapse/storage/media-store/ --ignore-existing` and a new script that basically just runs

```
- name: Ensure media store permissions are correct (generic case)
  file:
    path: "{{ matrix_synapse_media_store_path }}"
    owner: "{{ matrix_user_username }}"
    group: "{{ matrix_user_username }}"
    recurse: yes
  when: "not matrix_s3_media_store_enabled"
```

**Comment by spantaleev** [comment]
Maybe we can mention an alternative way to "restore media store" in `docs/restoring-media-store.md`.
One that would be beneficial if:
- the media store is very large and downloading it locally is not viable or too slow
- "locally" refers to a Mac filesystem, which has case-sensitivity troubles and might corrupt the media store hierarchy 

Those people can follow this alternative flow:
- (prerequisite, as in all cases) running a regular install (`--tags=setup-all`)
- `rsync`-ing the files to `/matrix/synapse/storage/media-store`
- fixing permissions using `chown -R matrix:matrix /matrix/synapse/storage/media-store`

By the way, using `--ignore-existing` would not be necessary in those cases. Seems like it's only beneficial if it got corrupted, then imported.. and then one is trying to fix it up from a non-corrupted backup.

**Comment by spantaleev** [comment]
I believe this is solved by the changes in 76506f34e0827650afb.
